src/lib/stack-client.js

```diff
@@ -65,7 +85,7 @@ const init = function(options) {
   cozyClient = options.cozyClient
   const legacyOptions = {
     ...options,
-    cozyURL: cozyClient.getStackClient().uri,
+    cozyURL: getCozyURLOrigin(),
```

Comment:
getCozyURLOrigin could take the `stackClient` so that the call to getStackClient is not duplicated.

Reply:
it was not true at stage 2 (getCozyURLOrigin did not exists) and won't be true at stage 12 (legacy options do not exist anymore). I'd prefer not to change too many temporary things